ecl/eclccserver/eclccserver.cpp

```diff
@@ -864,6 +874,17 @@ class EclccCompileThread : implements IPooledThread, implements IErrorReporter,
             workunit.clear();
             return;
         }
+
+        if (isContainerized())
+        {
+            if (config->getPropBool("@k8sJob"))
+                addTimeStamp(workunit, SSTcompilestage, "compile", StWhenK8sStarted, 0);
+            else
+                addTimeStamp(workunit, SSTcompilestage, "compile", StWhenDequeued, 0);
+        }
+        else
+            addTimeStamp(workunit, SSTcompilestage, "compile", StWhenDequeued, 0);
```

Comment:
May be simpler and avoids duplicate code to do it like this:

`        if (isContainerized() && config->getPropBool("@k8sJob"))
            addTimeStamp(workunit, SSTcompilestage, "compile", StWhenK8sStarted, 0);
        else
            addTimeStamp(workunit, SSTcompilestage, "compile", StWhenDequeued, 0);`

Reply:
minor, but yes I agree, but not worth changing now (merged)